imsim/readout.py

```diff
@@ -286,6 +287,11 @@ def get_primary_hdu(eimage, lsst_num, camera_name=None,
         phdu.header['RA'] = ratel
         phdu.header['DEC'] = dectel
         phdu.header['ROTCOORD'] = 'sky'
+        telcode = 'MC'
+    dayobs = eimage.header['DAYOBS']
+    seqnum = eimage.header['SEQNUM']
+    contrllr = eimage.header['CONTRLLR']
+    phdu.header['OBSID'] = f"{telcode}_{contrllr}_{dayobs}_{seqnum:06d}"
```

Comment:
I'm surprised `{dayobs}` isn't `{dayobs:03d}`, but since this was working before I assume that's right.

Reply:
nvm.  it's always 8 digits isn't it.  (not 3!)